packages/apollo-env/clone-types.js

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
I'm not sure where this belongs, but I don't think `apollo-env` is quite right. We might want a top-level `scripts` folder for these cross-platform scripts that we'll need to leverage.

Is typescript an option here?

Reply:
I'm not sure it'd fit outside anywhere though. It's something specifically used for env and in no other packages. I'd feel better hoisting it up a level if it was used anywhere else